### ISSUE TITLE
Missing indent line and reoccurring message regarding “nvim_tree_gitignore” keep persisting after update

### ISSUE THREAD

**Issue by mondenoir** [kyazdani42/nvim-tree.lua#812]
After a recent update, “nvim-tree” began the following 2 odd phenomenons:

1. A Reoccurring message pops up every single time I load up Noevim

<img width="713" alt="Screen Shot 2021-11-30 at 8 51 01 PM" src="https://user-images.githubusercontent.com/35069772/144055876-71b5a06d-2c87-477a-a0db-8119f2b0f36e.png">

2. The most outer indent line is missing and cursor is now hovering over the icon instead

<img width="168" alt="Pasted Graphic" src="https://user-images.githubusercontent.com/35069772/144056029-86581bb5-d5c2-43a2-b942-edf881e38921.png">

I tried to troubleshoot by doing the following to no avail:

1. Updated Neovim to nightly 0.6.0
2. Removing any “nvim_tree_gititgore” related options in my config file which is very minimal

<img width="291" alt="Pasted Graphic 1" src="https://user-images.githubusercontent.com/35069772/144056167-1465f377-ea0e-4788-9e68-af7998c81acd.png">

Has anyone encounter these problems after recent update? If so, please kindly share your solution.

**Comment by kyazdani42** [comment]
remove all `nvim_tree_gitignore` which is now set in `git = { ignore = true|false }` in the setup call

**Comment by mondenoir** [comment]
@kyazdani42 Thanks for the reply. I've added `git = { ignore = true|false }` to my config file as followed: 

<img width="284" alt="Screen Shot 2021-12-07 at 12 23 11 AM" src="https://user-images.githubusercontent.com/35069772/144883844-a5a2c928-6d8c-48f3-8b46-7bab69fc691e.png">

The same message still persist after restart along with the missing outer indent-line. More so, I've also try uninstalling/installing to no avail as well. Any idea why this is happening?

<img width="705" alt="Screen Shot 2021-12-07 at 12 24 56 AM" src="https://user-images.githubusercontent.com/35069772/144884007-dacc192b-bb45-4567-a58c-75da067c886e.png">

**Comment by kyazdani42** [comment]
did you remove the option `nvim_tree_gitignore` from your config ? Maybe if you are using something like packer, run `PackerCompile` ?

**Comment by mondenoir** [comment]
I truly appreciate your reply `kyazdani42` and believe me I've tried everything including `PackerCompile`,  uninstalling/installing, an empty config file, but the message always appears. It just puzzles me as to why the message would even appear if my config file is empty. You've made a great plugin but I guess I just have to use it as is for now

**Comment by kyazdani42** [comment]
this might be loaded somewhere in your config i guess. Try to `:let g:nvim_tree_gitignore` and see if it returns something, if it does, you still have this option somewhere in your config

**Comment by mondenoir** [comment]
You are right kyazdani42 the command you recommended did return something and I was able to track it down and resolve the error message issue :) Now, I just have to figure out how to resolve the indent line problem:

Before update:
<img width="181" alt="Screen Shot 2021-12-13 at 10 56 09 AM" src="https://user-images.githubusercontent.com/35069772/145748787-fc619ca2-6f6b-47c5-98e4-78b9e2005aba.png">

After update:
<img width="198" alt="Screen Shot 2021-12-13 at 10 52 48 AM" src="https://user-images.githubusercontent.com/35069772/145748832-c6bc60c1-96a7-468f-8a54-80299b805e87.png">

After some digging, I was able to get the indent line back by swapping the following file:` lua/nvim-tree/renderer/init.lua` to the previous version. There were 5 changes made on the current version located in line 106, 167, 183, 244, and 382. I've try commenting out those lines but to no avail. Nonetheless, I want to sincerely thank you for your replies and it's just very rare to have a creator of a plugin, that's so widely use, be so responsive to its users 🙏🙏🙏
